bgpd/bgp_attr.c

```diff
@@ -1635,18 +1635,17 @@ int bgp_mp_reach_parse(struct bgp_attr_parser_args *args,
 
 	/* Nexthop length check. */
 	switch (attr->mp_nexthop_len) {
+	case BGP_ATTR_NHLEN_VPNV4:
+		stream_getl(s); /* RD high */
+		stream_getl(s); /* RD low */
+		/* NOTE: intentional fall through - for consistency in rx processing */
```

Comment:
This line is beyond 80 columns.  Let's break it up

Reply:
*really* . Sigh. will do